src/pages/Admin/Venue/DetailsValidationSchema.ts

```diff
@@ -86,7 +86,10 @@ export const validationSchema = Yup.object()
     logoImageFile: createFileSchema("logoImageFile", false).notRequired(),
 
     showGrid: Yup.bool().notRequired(),
-    columns: Yup.number().notRequired().min(1).max(100),
+    columns: Yup.number().when("showGrid", {
+      is: true,
+      then: Yup.number().required("At least 5 columns are required").min(5),
+    }),
```

Comment:
Previously there was a maximum set in this validation, yet that doesn't seem to exist here anymore?